test/playwright/e2e/search-navigation.spec.ts

```diff
@@ -1,6 +1,6 @@
-const { expect, test } = require('@playwright/test')
+import { expect, test } from '@playwright/test'
 
-const { openFilters } = require('./utils')
+import { openFilters } from '~~/test/playwright/utils/navigation'
```

Comment:
This file should probably also just use `mockProviderApis` on lines 5-6

Reply:
Done in https://github.com/WordPress/openverse-frontend/pull/1153/commits/a3323af8af9d769e4beb112498a57aaccbd6cc6f